pages/Developer/API/websockets/apicommands.mdx

```diff
@@ -71,14 +71,43 @@ _Deletes the overlay that has ownership of the window toolbar._
 ***
 ### UpdateMediaPlayerInformation
 _Updates the media player information with an XSOMediaObject._
+```json filename="XSOMediaObject Object"
+{
+    "title": string, 
+    "album": string, 
+    "artist": string, 
+    "albumArt": string, // base64 encoded image
+    "applicationName": string, // spotify, youtube, etc
+    "playbackStatus": string // "playing" or "paused" 
+}
+```
 
 ***
 ### SetXSOverlaySystemSetting
-_Sets a user settings._
+_Sets a user settings. You can find the names in the settings and what they do with the GetSettings return object._
+```json filename="Setting Object"
+{
+    "internalName": string,
+    "value": object // type depends on setting
+}
+```
 
 ***
 ### SetActiveOverlaySetting
-_Sets a setting on the currently active overlay._
+_Sets a setting on the currently active overlay. Utilizes the setting object. Each overlay has these options._
+
+| Property | Type | Description | 
+| --- | --- | --- |
+| `pin` | `bool` | When enabled, the overlay is pinned. |
+| `autoHide` | `bool` | When enabled, the overlay will auto hide when not within the clipping angle |
+| `lock` | `bool` | When enabled, the overlay lock and will not be able to be moved or adjusted |
+| `opacity` | `float` | Opacity of the overlay, between 0 and 1 |
+| `blockInput` | `bool` | When enabled, the overlay will not accept inputs.|
+| `brightness` | `float` | Brightness of the overlay, between 0 and 1 |
+| `maxFPS` | `int` | Maximum FPS of the overlay when not active |
+| `minFPS` | `int` | Minimum FPS of the overlay when not active |
```

Comment:
Minimum and maximum FPS of overlays is just in general, not only when not active.

Generally, overlays that are being interacted with (ie being touched by the vr cursor) or animated in some way get boosted to the max fps, otherwise, they will stay at the minimum.

Reply:
Ah, I understand now.  I have it updated in this commit:
https://github.com/Xiexe/XSOverlay_Documentation/pull/2/commits/7a6137c711e876c4dbd6c0a0e44e6bb9b11c9e45